client/lib/i18n-utils/switch-locale.js

```diff
@@ -521,10 +521,14 @@ export function switchWebpackCSS( isRTL ) {
 		}
 
 		const newLink = await loadCSS( newHref, currentLink );
```

Comment:
@arthur791004 I'm placed the check after the promise `loadCSS` because I think the race condition issue happens because the RTL changes while the CSS files are being loaded. 

Do you think I should create another PR to try to `return` before `loadCSS` if the RTL has changed?

Reply:
Sounds good 👍 I have added the changes to this PR

Reply:
One issue that persists on the developer environment and calypso.live is duplicate entries in the `<head>` tag. Can we fix this?
![Duplicate CSS entries](https://user-images.githubusercontent.com/31164683/179032422-77ed5a9b-5f41-4b44-b21d-bfffbe334d0f.png)

Reply:
Thank you @d-alleyne, I've just fixed that issue in my last commit.